core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -12942,4 +12943,47 @@ private String toSqlWithBloat(RelNode root, int bloat) {
     assertThat(toSql(root, DatabaseProduct.CALCITE.getDialect()), isLinux(expectedSql));
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedBiqQuery));
   }
+
+  @Test public void testJSON_OBJECT() {
+    final RelBuilder builder = relBuilder();
+
+    Map <String, String> obj = new HashMap<>();
+    obj.put("Name", "John");
+    obj.put("Surname", "Mark");
+    obj.put("Age", "30");
+    List<RexNode> operands = new ArrayList();
+
+    for (Map.Entry<String, String> m : obj.entrySet()) {
+      operands.add(builder.literal(m.getKey()));
+      operands.add(builder.literal(m.getValue()));
+    }
+
+    final RexNode jsonNode = builder.call(SqlLibraryOperators.JSON_OBJECT, operands);
+
+    final RelNode doyRoot = builder
+        .scan("EMP")
+        .project(jsonNode)
+        .build();
+
+    final String expectedMONBiqQuery = "SELECT JSON_OBJECT('Surname', 'Mark', 'Age', '30', " +
```

Comment:
change this to `expectedBiqQuery`

Reply:
changed expectedMONBiqQuery to expectedBiqQuery